src/utils/helpers.py

```diff
@@ -1,7 +1,53 @@
 import json
+import logging
 from pathlib import Path
 
 
+def create_python_logger(
+    name: str, log_file_path: str = "/tmp/logs/spark.log"
```

Comment:
The only part I'm unsure about now is that the log file path is specified in like 3 different places: here, the Spark logger, and the CloudWatch upload. I couldn't think of a way to share the file path across all the modules nicely. Any thoughts?

Reply:
**[Suggestion, non-blocking]** I think it's totally fine! As I see it, we're basically just treating `/tmp/logs/spark.log` as the common interface for logs across services, so I think it makes sense that we would reference it in each place where we're making use of that interface. That being said, we could centralize things a tiny bit more if we defined a constant `SPARK_LOG_FILE = /tmp/logs/spark.log` in this file and then imported it into the `utils.spark` and `submit_jobs` modules -- that way if we ever want to change the logfile location we only have to do it once.

Reply:
Done in 316ddea!